### ISSUE TITLE
Model transformation problem

### ISSUE THREAD

**Issue by BoSeal** [DrSleep/tensorflow-deeplab-lfov#24]
Hello, can the tensorflow-based model be trained on the caff-based pre-training model?As a beginner, I am not clear about this question, can you answer for me?
![图片](https://user-images.githubusercontent.com/39556913/71308203-7cdd5200-2434-11ea-83ec-5fe068ffbe66.png)

**Comment by DrSleep** [comment]
yes, it can be if you first convert the weights of the caffe model into the format that tensorflow understands. All weights (regardless of the framework) are simply numeric arrays, the difference lies in how the frameworks store them.
